### PR TITLE
Added methods for inbound guidance

### PR DESCRIPTION
**Pull request by jkim65537**
I added these two calls as methods in your API. 

They are almost exact clones of your get_prep_instructions_{identifier} methods but they fetch inbound guidance instead. 

Below is a screenshot of methods being used in practice. I already packaged it (not included in PR) and am using it in my workflow.
=
screen shot 2019-03-08 at 10 59 34 am

http://docs.developer.amazonservices.com/en_US/fba_inbound/FBAInbound_GetInboundGuidanceForSKU.html

http://docs.developer.amazonservices.com/en_US/fba_inbound/FBAInbound_GetInboundGuidanceForASIN.html

### CODE REVIEW

**Comment by jameshiew** [comment]
Thanks for opening this PR, we currently have these methods in the `develop` branch, though some of the functionality in your version looks useful (e.g. the use of `utils.unique_list_order_preserved` to prevent an error). Would you mind rebasing the PR against `develop` branch? Currently in the process of figuring out what the best next steps may be for the `master`/`develop` branch in general.

File: mws/mws.py [hunk 1]
```diff
@@ -1121,6 +1121,46 @@ def update_inbound_shipment(self, shipment_id, shipment_name,
             ))
         return self.make_request(data, method="POST")
 
+    def get_inbound_guidance_for_sku(self, skus=None, marketplace_id=None):
+        """
+        Returns inbound guidance for a list of items by Seller SKU
+        to help you decide if an item should be sent to a given marketplace.
+        """
+        marketplace_id = marketplace_id or 'ATVPDKIKX0DER'
```
Comment (jameshiew) [inline mws/mws.py]: We probably shouldn't default the `marketplace_id` to 'ATVPDKIKX0DER' in the library as we don't do it elsewhere in the code

File: mws/mws.py [hunk 2]
```diff
@@ -1121,6 +1121,46 @@ def update_inbound_shipment(self, shipment_id, shipment_name,
...
+        skus = utils.unique_list_order_preserved(skus)
+
+        data = dict(
+            Action='GetInboundGuidanceForSKU',
+            MarketplaceId=marketplace_id,
+        )
+        data.update(utils.enumerate_params({
+            'SellerSKUList.ID.': skus,
+        }))
+        return self.make_request(data, method="POST")
```
Comment (jameshiew) [inline mws/mws.py]: I'm interested in `method="POST"`, the version in `develop` branch uses GET and it looks like this is a GET HTTP method. Does using `method="POST"` work for you?

File: mws/mws.py (hunk 1 above)
Comment (jkim65537) [reply mws/mws.py]: Those lines are removed.

File: mws/mws.py (hunk 2 above)
Comment (jkim65537) [reply mws/mws.py]: Using method="POST" works for me.

Comment (Bobspadger) [reply mws/mws.py]: The Amazon docs make no mention of this being a POST request either.

**Comment by elcolumbio** [comment]
i can't see the conflicts.
One thing you can do delete the file you created:  .DS_Store

File: mws/mws.py (hunk 2 above)
Comment (Bobspadger) [reply mws/mws.py]: @jkim65537 could you try the method as a GET to see if it works please?

**Comment by GriceTurrble** [comment]
As mentioned previously, I'd like to see these changes applied to our `develop` branch, which has a different structure than `master`, and finessing this PR into one that works for `develop` would take more effort than it is worth.

Please submit a new PR with our `develop` version as the base, applying your changes to `mws/api/feeds.py`.

**Pull request closed without merging**